### PR TITLE
PRO-6069: Tests optimisation

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-6069

### Change description ###
Optimisation of component and unit tests

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: app/resources/en/translation/addressLookup.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
         "message": "Please enter a postcode"
       },
       "invalid": {
-        "summary": "Postcode is invalid",
-        "message": "Postcode is invalid"
+        "summary": "You entered an invalid postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/addressLookup.json]: Revert these changes please

File: test/util/TestWrapper.js [hunk 2]
```diff
@@ -126,58 +126,61 @@ class TestWrapper {
...
     substituteErrorsContent(data, contentToSubstitute, type) {
-        forEach(Object.entries(contentToSubstitute), ([contentKey, contentValue]) => {
-            forEach(Object.entries(contentValue[type]), ([errorMessageKey, errorMessageValue]) => {
-                forEach(errorMessageValue.match(/\{(.*?)\}/g), (placeholder) => {
-                    const placeholderRegex = new RegExp(placeholder, 'g');
-                    contentToSubstitute[contentKey][type][errorMessageKey] = contentToSubstitute[contentKey][type][errorMessageKey].replace(placeholderRegex, data[placeholder]);
-                });
+        Object.entries(contentToSubstitute).forEach(([contentKey, contentValue]) => {
+            Object.entries(contentValue[type]).forEach(([errorMessageKey, errorMessageValue]) => {
+                const errorMessageValueMatch = errorMessageValue.match(/{(.*?)}/g);
```
Comment (douglasrice) [inline test/util/TestWrapper.js]: Just to confirm - there will only every be a single errorMessageValue for contentValue[type]
As previous code reads that there could be multiple error messages for contentValue

File: app/resources/en/translation/addressLookup.json (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply app/resources/en/translation/addressLookup.json]: Done

File: test/util/TestWrapper.js (hunk 2 above)
Comment (LucaDelBuonoHMCTS) [reply test/util/TestWrapper.js]: Not sure what you're saying here, the substance of the code hasn't changed here :-/
